library/Icingadb/Widget/Detail/CheckStatistics.php

```diff
@@ -132,18 +125,52 @@ protected function assembleBody(BaseHtmlElement $body)
                     : (new EmptyState(t('n. a.')))->setTag('span')
             )
         );
+
+        if ($this->isChecksDisabled()) {
+            $markerLast->addAttributes([
+                'class' => 'state-' . $this->object->state->getStateText()
+            ]);
```

Comment:
I don't think I talked about this change with @flourish86, that's just an artifact in the mockup. I see absolutely no reason to introduce this change here, it has nothing to do with active/passive checks being off.